app/models/compute_resources/foreman/model/openstack.rb

```diff
@@ -162,6 +162,10 @@ def create_vm(args = {})
       # fix internal network format for fog.
       args[:nics].delete_if(&:blank?)
       args[:nics].map! { |nic| nic.is_a?(String) ? { 'net_id' => nic } : nic }
+
+      # Process interface attributes to add fixed IP addresses
+      process_fixed_ips(args) if args[:interfaces_attributes].present?
```

Comment:
```suggestion
      process_fixed_ips(args)
```

There is another `return unless interfaces_attrs&.any?` check in the method itself